corehq/apps/hqcase/_design/views/by_domain_hq_user_id/map.js

```diff
@@ -1,5 +1,5 @@
 function(doc){
     if(doc.doc_type == "CommCareCase" && doc.hq_user_id) {
-        emit([doc.domain, doc.hq_user_id], null);
+        emit([doc.domain, doc.hq_user_id, doc.type], null);
```

Comment:
Now that we have to re index we may want to move this to a different design doc. This way in the future if we modify the view it doesn't cause all other views in the design doc to reindex. This was one of the takeaways from talking with cloudant